<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectWorkFlowDialog</class>
 <widget class="QDialog" name="ProjectWorkFlowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Project Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>471</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Observation :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Experience :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonExit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>550</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAjouter">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>480</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSupprimer">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>480</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonModifier">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>291</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonImport">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
